pandas/sparse/series.py

```diff
@@ -654,6 +656,48 @@ def combine_first(self, other):
         dense_combined = self.to_dense().combine_first(other)
         return dense_combined.to_sparse(fill_value=self.fill_value)
 
+    def to_coo(self, ilevels=(0,), jlevels=(1,), sort_labels=False):
```

Comment:
What about using `row_levels` and `column_levels` here instead of `ilevels`/`jlevels`? I like more explicit names.